packer/sea-starter-kit-win2019-eval.pkr.hcl

```diff
@@ -0,0 +1,280 @@
+  disk_size        = "${var.disk_size}"
+  floppy_files     = ["${path.root}/mnt/Autounattend.xml"]
+  headless         = "${var.headless}"
+  iso_checksum     = "${var.iso_checksum}"
+  iso_url          = "${var.iso_url}"
+  memory           = "${var.memory}"
+  shutdown_command = "${var.shutdown_command}"
+  switch_name      = "${var.vm_switch}"
+  vm_name          = "${var.vm_name}"
+  winrm_password   = "${var.password}"
+  winrm_timeout    = "10000s"
+  winrm_username   = "${var.user_name}"
+  output_directory = "${var.distribution_directory}"
+}
+
+build {
+  sources = ["source.hyperv-iso.sea-starter-kit"]
+
+  provisioner "comment" {
+    comment     = "Copying ${var.archive_name}.zip to c:/temp"
+    ui          = true
+    bubble_text =  false
+  }
+
+  provisioner "file" {
+    destination = "c:/temp/"
+    sources     = [
+        "${path.root}/build/${var.archive_name}.zip",
+        "${path.root}/build/${var.web_api}.zip",
+        "${path.root}/build/${var.admin_app}.zip",
+        "${path.root}/build/${var.swagger_ui}.zip",
+        "${path.root}/build/${var.databases}.zip"
+    ]
+  }
+
+  provisioner "comment" {
+    comment     = "Exctacting ${var.archive_name}.zip to c:/temp/${var.archive_name}"
+    ui          = true
+    bubble_text = false
+  }
+
+  provisioner "powershell" {
+    debug_mode        = "${var.debug_mode}"
+    elevated_password = "vagrant"
+    elevated_user     = "vagrant"
+    inline            = [
+        "Set-ExecutionPolicy bypass;",
+        "Set-Location c:/temp;",
+        "Expand-Archive ./${var.archive_name}.zip -Destination ./${var.archive_name}"
+      ]
+  }
+
+  provisioner "comment" {
+    comment          = "Executing c:/temp/${var.archive_name}/server-setup.ps1"
+    ui               = true
+    bubble_text      = false
+  }
+
+  provisioner "powershell" {
+    debug_mode        = "${var.debug_mode}"
+    elevated_password = "vagrant"
+    elevated_user     = "vagrant"
+    inline            = ["Set-ExecutionPolicy Bypass", "Set-Location c:/temp/${var.archive_name}/scripts", "./server-setup.ps1"]
+  }
+
+  provisioner "comment" {
+    comment     = "Server Setup complete. Restarting...."
+    ui          = true
+    bubble_text =  false
+  }
+
+  provisioner "windows-restart" {
+    restart_check_command = "powershell -command \"& {Write-Output 'Server Setup complete. Restarting....'}\""
+  }
+
+  provisioner "comment" {
+    comment     = "Installing Required Powershell Packages"
+    ui          = true
+    bubble_text =  false
+  }
+
+  provisioner "powershell" {
+    debug_mode        = "${var.debug_mode}"
+    elevated_password = "vagrant"
+    elevated_user     = "vagrant"
+    inline            = [
+        "Set-ExecutionPolicy Bypass",
+        "Set-Location c:/temp",
+        "Install-PackageProvider -Name NuGet -MinimumVersion \"2.8.5.201\" -Scope AllUsers -Force",
+        "Install-Module -Name SqlServer -MinimumVersion \"21.1.18068\" -Scope AllUsers -Force -AllowClobber"
```

Comment:
if we just set the package source here we can let the database-management.psm1 setup the sql server module, and then we don't have to worry about keeping the version in sync

Reply:
removed from scripts, and moved the package provider install up to earlier in the process